app/services/container_topology_service.rb

```diff
@@ -80,10 +80,10 @@ def entity_display_type(entity)
           "Container" # the container entity itself
         end
       else
-        if name.eql? "Vm"
-          'VM'
+        if entity.kind_of?(Vm)
+          name.upcase # turn Vm to VM because it's an abbreviation
```

Comment:
Thanks... I know I'm not looking at it... I just know solving the same problem differently is hard to maintain in the long run.  As long as we plan to look a related things like this... I'm good.  I just try to call them when I see them.  Once things are merged, we don't look.